### ISSUE TITLE
When the hidden dim is changed from 256 to 384, the model cannot be trained.

### ISSUE THREAD

**Issue by Lotus-95** [fundamentalvision/Deformable-DETR#83]
In this case, the loss will drop to 24 and then no longer continue to drop, the map is always 0. Even changing the learning rate to 5e-5 won’t work(In this case, loss will converge to near 14).
